onnx2kerastl/converter.py

```diff
@@ -197,7 +197,10 @@ def onnx_to_keras(onnx_model, input_names,
                         node_params['is_embedding'] = True
                     layers[node_input] = weights[node_input]
                 else:
-                    raise AttributeError('Current node is not in weights / model inputs / layers.')
+                    if node_input == "" and (node_type == 'Pad' or node_type == 'Resize'):
```

Comment:
can you explain? how its possible

Reply:
Some layers could have empty inputs in its strings. Adding an Onnx example

Reply:
![image](https://user-images.githubusercontent.com/95576526/193601415-61eb6613-28a8-41c5-a808-33e0730cda69.png)
Here, for example, the constant_value param has no name